lib/matplotlib/tests/test_colors.py

```diff
@@ -1301,6 +1301,48 @@ def test_to_rgba_array_alpha_array():
     assert_array_equal(c[:, 3], alpha)
 
 
+@pytest.mark.parametrize('rgba_alpha',
+                         [('red', 0.5), ('#ff0000', 0.5), ('#f00', 0.5),
+                          ('#ff0000ff', 0.5), ((1.0, 0.0, 0.0, 1.0), 0.5)])
+def test_to_rgba_array_color_alpha_tuple(rgba_alpha):
+    """to_rgba_array() accepts tuple (color, alpha) argument."""
+    booleans = mcolors.to_rgba_array(rgba_alpha) \
+               == mcolors.to_rgba_array('red', alpha=0.5)
+    assert booleans.all()
```

Comment:
You are only testing single colors as input. Please add at least one multi-color input (and maybe not as many single ones - while these are different in the lower to_rgba handling, they all follow the same code path in to_rgba_array()).